.github/workflows/integration.yaml

```diff
@@ -209,6 +209,9 @@ jobs:
 
   e2e-test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        k8s: [v1.28.13, v1.29.8, v1.30.4, v1.31.1]
```

Comment:
It doesn't look like Kind publishes image tags without the minor version e.g. `v1.31` so I had to grab the latest patch versions manually from https://hub.docker.com/r/kindest/node/tags. Is there a way to make this Dependabot compatible or should we simply bump all things that are related to Kubernetes versions manually on a schedule e.g. once a month?

Reply:
I think it's fine to specify the k8s version manually for now. I am not sure whether there is a a way to automatically update the k8s version matrix.

Reply:
Yeah I couldn't find any examples of doing this automatically. Let's just do it manually.